uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -472,11 +484,19 @@ define([
       /**
        * Flattens statuses from object to array of objects. This is used to
        * populate the dropdown with array of statuses.
+       * Also it checks if given status is available to manager. If manager applies leave
+       * on behalf of staff then cancelled is also removed from her list of available statuses.
        *
        * @return {Array}
        */
       this.getStatuses = function () {
-        return Object.values(this.requestStatuses);
+
+        return _.filter(Object.values(this.requestStatuses), function (status) {
```

Comment:
given what you're doing in the callback, probably `_.reject` is more appropriate